### PR TITLE
RecipeFeed, ReviewFeed, ReviewFocus

### PR DESCRIPTION
**Pull request by MrNiceRicee**
- [x] fixed some yarn issues
- [x] recipe feed routes to recipe page
- [x] recipe feed routes to recipe review list   
- [x] styled recipe feed
- [x] styled recipe card
- [x] styled review feed
- [x] styled review card 
- [x] added review focus
- [x] added review focus comments
- [ ] the pages need to receive data from the backend. And as an effect, also be able to route to the correct pages. The routing are just mock ups
- moved recipe feed to the proper folder (social folder)
- implemented sort in both Recipe Feed and Recipe Review list
- fixed linting problems
- Added a new package, so people pulling down has to re run `yarn`, and also `yarn run eslint --init` for eslint to re run (if it's not working)

### CODE REVIEW

File: src/component/App.jsx [hunk 1]
```diff
@@ -120,9 +121,11 @@ function App() {
                     <RecipeFocusStart />
                   </Route>
                   <Route exact path="/recipe/:recipeId/reviews">
-                    Reviews
                     <ReviewList />
                   </Route>
+                  <Route exact path="/recipe/:recipeId/review/:reviewId">
```
Comment (aleceche) [inline src/component/App.jsx:126]: should the end of this path be recipes/:recipeId/reviews/:reviewId?
Probably not necessary, but might be more clear

File: src/component/social/recipefeed/ListOfRecipes.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import {
+  Box,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+import RecipeCard from './RecipeCard';
+
+const listOfRecipes = ({ data }) => (
```
Comment (aleceche) [inline src/component/social/recipefeed/ListOfRecipes.jsx:10]: should start with a capital letter?

File: src/component/social/recipefeed/RecipeFeedStart.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+const RecipeFeed = () => {
+  // user data will be data about recipes and display the title, body and favorite/#ofreviews
+  // be able to filter recipe info from select tag by most liked or most commented or my reviews
+  const filterOptions = [
+    'Most Reviews',
+    'Most Favorites',
+  ];
+
+  const test = [
```
Comment (aleceche) [inline src/component/social/recipefeed/RecipeFeedStart.jsx:18]: ew hard coded data

File: src/component/social/review/ReviewCard.jsx [hunk 4]
```diff
@@ -1,47 +1,94 @@
...
 
-const ReviewCard = (props) => {
-  let handleUpvote = () => {
+import ShowMoreText from 'react-show-more-text';
+import SpacingDesign from '../../context/design/SpacingDesign';
+
+const ReviewCard = ({
+  id, title, authorName, body, upvotes, downvotes, recipeId, date,
+}) => {
+  const handleUpvote = () => {
```
Comment (aleceche) [inline src/component/social/review/ReviewCard.jsx:22]: I assume these are to update the RTDB?

File: src/component/social/review/ReviewFocus.jsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+} from '@material-ui/core';
+import { ThumbUp, ThumbDown } from '@material-ui/icons';
+import { Link } from 'react-router-dom';
+import Image from 'material-ui-image';
+
+import SpacingDesign from '../../context/design/SpacingDesign';
+import ReviewFocusComment from './ReviewFocusComment';
+
+const ReviewFocus = () => {
+  useEffect(() => {
```
Comment (aleceche) [inline src/component/social/review/ReviewFocus.jsx:19]: what is this useEffect for?

File: src/component/social/review/ReviewFocusComment.jsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  Typography,
+  Button,
+  Card,
+} from '@material-ui/core';
+import { ThumbUp, ThumbDown } from '@material-ui/icons';
+
+import SpacingDesign from '../../context/design/SpacingDesign';
+
+const ReviewFocusComment = () => {
+  useEffect(() => {
```
Comment (aleceche) [inline src/component/social/review/ReviewFocusComment.jsx:14]: ditto above?

**Review by aleceche — APPROVED** [review]

File: src/component/social/review/ReviewFocus.jsx (hunk 5 above)
Comment (MrNiceRicee) [reply src/component/social/review/ReviewFocus.jsx:19]: Place holder when we use Axios

File: src/component/social/review/ReviewFocusComment.jsx (hunk 6 above)
Comment (MrNiceRicee) [reply src/component/social/review/ReviewFocusComment.jsx:14]: Same thing as above, for axios

**Pull request merged**